tests/components/DirectAnswer.test.tsx

```diff
@@ -21,28 +21,8 @@ it('renders null when there is no direct answer in state', () => {
   expect(container).toBeEmptyDOMElement();
 });
 
-describe('FieldValue direct answer', () => {
+describe('Field value direct answer analytics', () => {
   beforeEach(() => mockState(fieldValueDAState));
-
-  it('title text', () => {
-    render(<DirectAnswer />);
-    const expectedTitle = '[entityName] / [fieldName]';
```

Comment:
do we need a test like this in the `FieldValueDirectAnswer` test suite?

Reply:
didn't think it's needed, these tests are basically testing UI changes, which the storybook stories should take care of already! As for the link test, that is added to FieldValueDirectAnswer test suite already in the pr